src/server/api/accounts/getAccountInstance.ts

```diff
@@ -0,0 +1,22 @@
+'use server'
```

Comment:
single quote here, another file has double. lets agree we stick with the double k?